src/index.js

```diff
@@ -48,6 +48,13 @@ function serviceFactory(swaggerConfigurator, req) {
           request.headers.correlationid = req.headers?.correlationid;
           newSpanLogger = req.gb?.logger?.loggerWithNewSpan?.();
           request.headers.span = newSpanLogger?.spanId;
+
```

Comment:
probably makes the code a little less verbose to do `const ua = request.headers['user-agent'];`